cmd/katapult/organizations.go

```diff
@@ -1,16 +1,23 @@
 package main
 
 import (
+	"context"
 	"fmt"
 
-	"github.com/krystal/go-katapult/core"
-
 	"github.com/krystal/go-katapult"
 
+	"github.com/krystal/go-katapult/core"
```

Comment:
Nitpick: I prefer grouping external imports together into a single section without blank lines, as I think it's a bit neater, specially once you end up with a lot of imports.